easybuild/easyconfigs/o/OpenFOAM/external-3rd-ThirdParty-2.3.1.patch

```diff
@@ -0,0 +1,66 @@
+#Use external CGAL and METIS (via CGAL_ROOT, and METIS_ROOT in easyconfig)
+#B. Hajgato OCT 23,2014
```

Comment:
@boegel: The patch was made that day... Sometimes it takes time to get from develop to master...
so what should be the date at the end?